lib/manager/components/ProjectSettingsForm.js

```diff
@@ -78,6 +78,21 @@ export default class ProjectSettingsForm extends MessageComponent<Props, State>
 
   componentWillMount () {
     this._updateStateFromProps(this.props)
+    window.addEventListener('keydown', this._handleKeyDown)
+  }
+
+  componentWillUnmount () {
+    window.removeEventListener('keydown', this._handleKeyDown)
+  }
+
+  _handleKeyDown = (e: SyntheticKeyboardEvent<HTMLInputElement>) => {
+    switch( e.keyCode ) {
+      case 13: // ENTER
+        this._onSaveSettings()
```

Comment:
There needs to be some kind of validation check done before calling `_onSaveSettings`.  It's possible to create a project without a name with this change.